File: MODUL-2/C/PRAK205-2310817220002-RaudatulSholehah.c
#include <stdio.h>
#include <math.h>
int main()
{
    int A, B;
    printf("Input\n");
    scanf("%d %d", &A, &B);

    int Alas = sqrt(pow(B, 2) - (pow(A, 2)));
    int Tinggi = A;
    int Keliling = A + B + Alas;
    int Luas = 0.5 * Alas * Tinggi;

    printf("Alas        = %d cm \n", Alas);
    printf("Tinggi      = %d cm \n", Tinggi);
    printf("Keliling    = %d cm \n", Keliling);
    printf("Luas        = %d cm^2 \n", Luas);
    return 0;
}

File: MODUL-2/Python/PRAK205-2310817220002-RaudatulSholehah.py
import math
print("input")
A = input().split()

if len(A)==1:
    A = int(A[0])
    B = int(input())

elif len(A)==2:
    A, B = map(int, A )

Alas = int(math.sqrt(B**2 - A**2))
Tinggi = A
Keliling = A + B + Alas
Luas = 0.5 * Alas * Tinggi

print("output")
print(f"Alas        = {Alas} cm")
print(f"Tinggi      = {Tinggi} cm")
print(f"Keliling    = {Keliling} cm")
print(f"Luas        = {Luas:.0f} cm^2")